liiatools/datasets/social_work_workforce/lds_csww_clean/converters.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import re
+
+log = logging.getLogger(__name__)
+
+
+def to_category(string, categories):
+    """
+    Matches a string to a category based on categories given in a schema file
+    the schema file should contain a dictionary for each category for this function to loop through
+    return blank if no categories found
+
+    :param string: Some string to convert into a category value
+    :param categories: A list of dictionaries containing different category:value pairs
+    :return: Either a category value, "error" if category is invalid or blank string
+    """
+    for code in categories:
+        if str(string).lower() == str(code["code"]).lower():
+            return code["code"]
+        if (
+            str(string).lower() == str(code["code"]).lower() + ".0"
+            ):  # In case integers are read as floats
+            return code["code"]
+        if "name" in code:
+            if str(code["name"]).lower() in str(string).lower():
+                return code["code"]
+            if not string:
+                return ""
+        elif not string:
+            return ""
+    return "error"
+
+
+def to_integer(value, config):
+    """
+    Convert any strings that should be integers based on the config into integers
+
+    :param value: Some value to convert to an integer
+    :param config: The loaded configuration
+    :return: Either an integer value or an "error" string if value could not be formatted as integer or a blank string if no value provided
+    """
+    if config == "integer":
```

Comment:
This is already checked before this function is being read in. I would probably opt for combining the to_integer and to_decimal functions (maybe to to_numeric) and keep the if statements in the new to_numeric function (removing them from clean_numeric)